packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -27,8 +27,40 @@
     min-height: 100vh;
   }
 
+  .nav {
+    @media screen and (max-width: $terra-medium-breakpoint) {
```

Comment:
I think this can always apply in every breakpoint. Also move the styles that are in nav.scss for background opacity upinto this class and delete it there.